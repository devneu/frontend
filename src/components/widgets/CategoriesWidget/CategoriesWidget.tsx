import React from 'react';
import { Button, Card, Col, Row } from 'antd';
import './CategoriesWidget.scss';

const CategoriesWidget = () => (
  <Card title="Categories" className="categories-widget-card">
    <Row justify="start" gutter={[8, 8]}>
      <Col span={12}>
        <Button type="link" href="#">
          Web Design
        </Button>
      </Col>
      <Col span={12}>
        <Button type="link" href="#">
          HTML
        </Button>
      </Col>
      <Col span={12}>
        <Button type="link" href="#">
          Freebies
        </Button>
      </Col>
      <Col span={12}>
        <Button type="link" href="#">
          JavaScript
        </Button>
      </Col>
      <Col span={12}>
        <Button type="link" href="#">
          CSS
        </Button>
      </Col>
      <Col span={12}>
        <Button type="link" href="#">
          Tutorials
        </Button>
      </Col>
    </Row>
  </Card>
);

export default CategoriesWidget;
