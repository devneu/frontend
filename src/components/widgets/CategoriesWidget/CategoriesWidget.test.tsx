import React from 'react';
import renderer from 'react-test-renderer';
import CategoriesWidget from './CategoriesWidget';

describe('CategoriesWidget', () => {
  it('Should successfully render', () => {
    const wrapper = renderer.create(<CategoriesWidget />).toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
