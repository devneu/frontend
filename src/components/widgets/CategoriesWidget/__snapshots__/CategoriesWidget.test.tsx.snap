// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CategoriesWidget Should successfully render 1`] = `
<div
  className="ant-card ant-card-bordered categories-widget-card"
>
  <div
    className="ant-card-head"
    style={Object {}}
  >
    <div
      className="ant-card-head-wrapper"
    >
      <div
        className="ant-card-head-title"
      >
        Categories
      </div>
    </div>
  </div>
  <div
    className="ant-card-body"
    style={Object {}}
  >
    <div
      className="ant-row ant-row-start"
      style={
        Object {
          "marginBottom": -4,
          "marginLeft": -4,
          "marginRight": -4,
          "marginTop": -4,
        }
      }
    >
      <div
        className="ant-col ant-col-12"
        style={
          Object {
            "paddingBottom": 4,
            "paddingLeft": 4,
            "paddingRight": 4,
            "paddingTop": 4,
          }
        }
      >
        <a
          className="ant-btn ant-btn-link"
          href="#"
          onClick={[Function]}
        >
          <span>
            Web Design
          </span>
        </a>
      </div>
      <div
        className="ant-col ant-col-12"
        style={
          Object {
            "paddingBottom": 4,
            "paddingLeft": 4,
            "paddingRight": 4,
            "paddingTop": 4,
          }
        }
      >
        <a
          className="ant-btn ant-btn-link"
          href="#"
          onClick={[Function]}
        >
          <span>
            HTML
          </span>
        </a>
      </div>
      <div
        className="ant-col ant-col-12"
        style={
          Object {
            "paddingBottom": 4,
            "paddingLeft": 4,
            "paddingRight": 4,
            "paddingTop": 4,
          }
        }
      >
        <a
          className="ant-btn ant-btn-link"
          href="#"
          onClick={[Function]}
        >
          <span>
            Freebies
          </span>
        </a>
      </div>
      <div
        className="ant-col ant-col-12"
        style={
          Object {
            "paddingBottom": 4,
            "paddingLeft": 4,
            "paddingRight": 4,
            "paddingTop": 4,
          }
        }
      >
        <a
          className="ant-btn ant-btn-link"
          href="#"
          onClick={[Function]}
        >
          <span>
            JavaScript
          </span>
        </a>
      </div>
      <div
        className="ant-col ant-col-12"
        style={
          Object {
            "paddingBottom": 4,
            "paddingLeft": 4,
            "paddingRight": 4,
            "paddingTop": 4,
          }
        }
      >
        <a
          className="ant-btn ant-btn-link"
          href="#"
          onClick={[Function]}
        >
          <span>
            CSS
          </span>
        </a>
      </div>
      <div
        className="ant-col ant-col-12"
        style={
          Object {
            "paddingBottom": 4,
            "paddingLeft": 4,
            "paddingRight": 4,
            "paddingTop": 4,
          }
        }
      >
        <a
          className="ant-btn ant-btn-link"
          href="#"
          onClick={[Function]}
        >
          <span>
            Tutorials
          </span>
        </a>
      </div>
    </div>
  </div>
</div>
`;
